prep-javascript-objects/main.js

```diff
@@ -0,0 +1,21 @@
+const person = {
+  firstName: 'Khanh',
+  lastName: 'Nguyen',
+  hobby: 'Building Computers',
+};
+
+console.log(person);
+
+const fullName = person.firstName + ' ' + person.lastName;
+
+console.log("The person's full name is :", fullName);
+
+const job = (person.job = 'Student');
+
+console.log("The person's current job is: ", job);
+
+const previousJob = (person['previousJob'] = 'Vendor Management');
```

Comment:
const previousJob = (person['previousJob'] = 'Vendor Management');

instead of using () around the expression, you can just do:

const previousJob = person['previousJob'] = 'Vendor Management';

Reply:
Thank you Brett!